js/components/popupWindow.js

```diff
@@ -69,35 +72,55 @@ class PopupWindow extends ImmutableComponent {
   }
 
   render () {
-    let styles = {}
+    let style = {}
     if (parseInt(this.width)) {
-      styles.width = (parseInt(this.width) + 2)
+      style.width = (parseInt(this.width) + 2)
     }
     if (parseInt(this.height)) {
-      styles.height = (parseInt(this.height) + 2)
+      style.height = (parseInt(this.height) + 2)
     }
     if (parseInt(this.top)) {
       if (this.top + this.height < window.innerHeight) {
-        styles.top = this.top
+        style.top = this.top
       } else {
-        styles.bottom = 0
+        style.bottom = 0
       }
     }
     if (parseInt(this.left)) {
       if (this.left + this.width < window.innerWidth) {
-        styles.left = this.left
+        style.left = this.left
       } else {
-        styles.right = 0
+        style.right = '1em'
       }
     }
 
     return <div
       className={cx({
         popupWindow: true,
-        reverseExpand: styles.right !== undefined
+        [css(styles.popupWindow)]: true,
+        [css((style.right !== undefined) && styles.reverseExpand)]: true
```

Comment:
@bridiver do you remember? https://github.com/brave/browser-laptop/commit/4bf6154f3748224c2e51a9a2f1c269af929bec65#diff-19f896aac3ef3233d5725e9e002625ccR19